app/models/invitation.rb

```diff
@@ -26,6 +26,7 @@ class InvitationAlreadyUsed < StandardError
   scope :pending, -> { not_cancelled.single_use.where(accepted_at: nil) }
   scope :shareable, -> { not_cancelled.where(single_use: false) }
   scope :single_use, -> { not_cancelled.where(single_use: true) }
+  scope :ignored, -> (send_count, age_in_hours) { pending.where('send_count = ? AND created_at < ?', send_count, age_in_hours.hours.ago) }
```

Comment:
I'd slightly prefer this to be

```
pending.where(send_count: send_count).where('created_at < ?', age_in_hours.hours.ago)
```

...I think in the interest of writing as little SQL as possible. 💛